### PR TITLE
chore: update scripts

### PR DESCRIPTION
**Pull request by jcs47**

### CODE REVIEW

File: join/config.toml [hunk 1]
```diff
@@ -396,28 +396,25 @@ namespace = "tendermint"
...
-# For testing, the connection to the btc bridge can be ignored.
-# In that case, all functionality requiring access to Bitcoin data is mocked.
-start-with-bridge = true
-
 ##### EVM bridges options #####
 [[axelar_bridge_evm]]
 
 # Chain name
 name = "Ethereum"
-# Address of the ethereum RPC proxy
```
Comment (Windforcer) [inline join/config.toml:416]: We should keep this comment, mirrors bitcoin comment and is consistent

**Review by Windforcer — APPROVED** [review]

File: join/config.toml (hunk 1 above)
Comment (jcs47) [reply join/config.toml:416]: done

File: join/config.toml [hunk 2]
```diff
@@ -396,28 +396,27 @@ namespace = "tendermint"
...
+
+# Address of the ethereum RPC server
+# chain maintainers must set their own rpc endpoint
 rpc_addr    = "<your-ethereum-rpc>"
```
Comment (ggutoski) [inline join/config.toml:416]: Why still `<your-ethereum-rpc>` and not the empty string?  Won't this will continue to cause confusion?

Comment (jcs47) [reply join/config.toml:416]: yeah, maybe we should have removed this as well. But for EVM at least we still have the `start-with-bridge` flag, which is currently set to false. So this shouldn't be a problem.

Comment (jcs47) [reply join/config.toml:416]: also, there is another thing to consider: bitcoin is currently inactive, so we don't even want validators to set up rpcs. This way we are encouraging them to set them up explicitly for ethereum. In any case, we might need to update this config file for more EVM chains, so we can get back to this later.

**Pull request merged**